Formula/i/imagineer.rb

```diff
@@ -0,0 +1,20 @@
+class Imagineer < Formula
+  desc "Image processing and conversion from the terminal"
+  homepage "https://github.com/foresterre/imagineer"
+  url "https://github.com/foresterre/imagineer/archive/refs/tags/v0.24.0.tar.gz"
+  sha256 "2511068e1919ee67af1fb9203a1bbcfc9f24894cd3b2e341c52d493e79535e93"
+  license any_of: ["Apache-2.0", "MIT"]
```

Comment:
maybe add head build as well?
```suggestion
  license any_of: ["Apache-2.0", "MIT"]
  head "https://github.com/foresterre/imagineer.git", branch: "main"
```